src/lib/log.ts

```diff
@@ -14,6 +15,36 @@ const filterRequestType = winston.format((info: any) =>
   info.type === 'request' ? false : info,
 )
 
+const getTransports = (): Transport[] => {
+  const transports: Transport[] = []
+
+  if (
+    process.env.NODE_ENV === 'development' ||
+    process.env.NODE_ENV === 'test'
+  ) {
+    transports.push(
```

Comment:
No need to mutate, just return the array here and below. Early return is better than `else if`/`else` IMO